<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeedModel</class>
 <widget class="QMainWindow" name="SpeedModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelOne">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>10</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButtonOne">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="menuOne"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="menuTwo"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionOne">
   <property name="text">
    <string>重来</string>
   </property>
  </action>
  <action name="actionTwo">
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="actionThree">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="menuOne">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="menuTwo">
   <property name="text">
    <string>查看游戏成绩</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="musicOne">
   <property name="text">
    <string>第一首</string>
   </property>
  </action>
  <action name="musicTwo">
   <property name="text">
    <string>第二首</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
